### DISCUSSION TITLE
Is it possible to export map from build version to package version of carle?

### DISCUSSION THREAD

**Discussion by mimits** [carla-simulator/carla#4830] [Q&A]
Hey guys just wondering, is it possible to create a package version of Carla from the built version of Carla( the one where you start with make launch and it opens unreal editor)
if not, how exactly can I export maps from the build version of Carla for the package version of Carla?

**Comment by Yurstan** [comment]
I ingested my_map (.fbx and .xodr obatined with OSM and Blender) in Carla builded from source (including UnrealEngine) and obtained the my_map.tar.gz. 

Moved this .tar.gz inside the /home/carla/Import folder of a Carla Docker container (which i assume to be the same as a package version) then i imported the my_map.tar.gz through the ImportAsset.sh script (no errors showed up); the script generated a new folder named my_map in the Carla directory (/home/carla).

Finally executed the "python3 config.py --map my_map" (present in /home/carla/PythonAPI/util) but it ends with "map not found"...executing "python3 config.py --list" confirms that my_map is not found (while i can see other Maps given with Carla).

Any suggestion?

**Comment by hotmedal** [comment]
I'm also stuck on this. I have made the map in blender, exported the fbx, converted the osm to xodr, got those both into source compiled carla unreal engine, saved the project as umap file. And now I don't know what to do. This umap file is exactly what I want in the packaged version but simply copying it over doesn't work. When I try to change the map using config.py, carla crashes with LowLevelFatalError and segfaults.

**Comment by fezsid** [comment]
Was anyone able to get the solution to this?

**Reply by hotmedal** [reply]
I have a tutorial that might help 
 https://nyu.zoom.us/rec/share/d_MXWFYyeC0fKqhP_1ugOv1W-njniYEi0BwXyqgTcB2zLe6fYrOsGLCZkyqRTt7e.NAdMO7tkCmO6XNbM
